app/Http/Controllers/api/Student/UpdateStudentProjectController.php

```diff
@@ -4,31 +4,40 @@
 
 namespace App\Http\Controllers\api\Student;
 
+use Exception;
 use App\Http\Controllers\Controller;
 use App\Service\Student\UpdateStudentProjectService;
 use Illuminate\Http\JsonResponse;
 use App\Http\Requests\UpdateStudentProjectRequest;
 use App\Exceptions\StudentNotFoundException;
 use App\Exceptions\ProjectNotFoundException;
+use App\Exceptions\UserNotFoundException;
+use App\Service\Student\StudentService;
 
 class UpdateStudentProjectController extends Controller
 {
-    private $studentupdateProjectService;
+    private UpdateStudentProjectService $updateStudentProjectService;
 
-    public function __construct(UpdateStudentProjectService $studentupdateProjectService)
+    private StudentService $studentService;
+
+    public function __construct(UpdateStudentProjectService $updateStudentProjectService, StudentService $studentService)
     {
-        $this->studentupdateProjectService = $studentupdateProjectService;
+        $this->updateStudentProjectService = $updateStudentProjectService;
+        $this->studentService = $studentService;
     }
 
     public function __invoke(UpdateStudentProjectRequest $request, $studentId, $projectId): JsonResponse
     {       
         try {
+            $userProfile = $this->studentService->findUserByStudentID($studentId);
```

Comment:
Passport check is redundant when the authorisation is required via a route middleware inclusion

Reply:
done